proto/regen/ecocredit/v1/tx.proto

```diff
@@ -273,6 +311,128 @@ message MsgCreateProjectResponse {
   string project_id = 1;
 }
 
+// MsgCreateUnregisteredProject is the Msg/CreateUnregisteredProject request type.
+message MsgCreateUnregisteredProject {
+  option (cosmos.msg.v1.signer) = "creator";
+
+  // creator is the address of the account creating the project that will become
+  // the admin of the project upon creation.
+  string creator = 1;
```

Comment:
Do we want to name this `admin` instead of `creator` to remain consistent with the language we're using in our existing framework?

Reply:
Sure